docs/modules/operation/pages/device-config-backup/introduction.adoc

```diff
@@ -0,0 +1,64 @@
+= Device Configuation Backup
+
+OpenNMS allows to backup the configuration of devices like routers or firewalls. OpenNMS and Minions provide a TFTP server. Devices send their configuration to that TFTP server. The backup can happen either pull or push based.
+
+* Pull based backup: OpenNMS uses its poller infrastructure to schedule tasks that connect via SSH into devices and execute configurable scripts that do the TFTP uploads.
+* Push based backup: Devices may upload their configuration by themselves. This may happen for example either by administrator interactions or by scheduled jobs running on the devices theirselves.
```

Comment:
```suggestion
* Push-based backup: Devices may upload their configuration by themselves. This may happen, for example, either by administrator interaction or scheduled jobs running on the devices themselves.
```